src/geo/transform.js

```diff
@@ -1417,15 +1417,56 @@ class Transform {
         return new LngLatBounds(new LngLat(west, south), new LngLat(east, north));
     }
 
+    // Gets minimum bounding lngLats for non-Mercator and non-globe projections.
+    _getBoundsProjection(): LngLatBounds {
```

Comment:
It seems like these names could use some renaming to denote what these functions are specialized for, I propose:

```
_getBoundsProjection -> _getBoundsNonRectangular
_getBounds -> _getBoundsRectangular
_getBounds3D -> getBoundsRectangular3D
_getBoundsGlobe -> leave as is
```